game/game/playercontrol.h

```diff
@@ -75,6 +75,68 @@ class PlayerControl final {
 
     using Action=KeyCodec::Action;
 
+    // Forward-Backward, Right-Left
+    static constexpr size_t NumAxes = 2;
+
+    struct AxisStatus { 
+
+        /// Main direction (e.g. W or Up arrow)
+        std::array<bool, 2> main;
```

Comment:
Should it be `NumAxes`, instead of 2? Or maybe something like `KeyCodec::NumMappings`?